BusyTiming/ActivityStruct.swift

```diff
@@ -5,16 +5,16 @@
 //  Created by Владимир Бажанов on 11/12/21.
 //
 
-import Foundation
+import RealmSwift
 
-struct Activity {
-    var name: String
-    var isSelected:  Bool
-    var time: Int
-    
-    init(nameInit: String, selectedInit:Bool, timeInit:Int) {
-      name = nameInit
-      isSelected = selectedInit
-      time = timeInit
+ class Activity: Object {
+     @objc dynamic var name: String = ""
+     @objc dynamic var startTime: Double = 0
+     @objc dynamic var endTime: Double = 0
+}
+
+extension Activity {
+    @objc dynamic var duration: Int {
```

Comment:
I belive it's not required to have it `@objc dynamic`. It's needed for Realm objects for ivars that are stored ib the DB, not for calculated properties.